Src/library/memory_analyzer/singletons/ma_icons_singleton.e

```diff
@@ -38,7 +38,10 @@ feature {NONE} -- Implementation
 			l_result: like internal_pixmap_path
 		do
 			l_result := internal_pixmap_path
-			check attached l_result end -- FIXME: Implied by ...?
+			if l_result = Void then
```

Comment:
Could we remove this and instead make the `set_pixmap_directory` a creation procedure that sets `internal_pixmap_path`?